src/main/java/games/strategy/debug/LoggingConfiguration.java

```diff
@@ -0,0 +1,49 @@
+package games.strategy.debug;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import java.util.logging.LogManager;
+
+import com.google.common.annotations.VisibleForTesting;
+
+/**
+ * Provides support for configuring the Java platform's core logging facilities.
+ */
+public final class LoggingConfiguration {
+  private LoggingConfiguration() {}
+
+  /**
+   * Initializes the application logging configuration.
+   *
+   * <p>
+   * The default application logging configuration is read from the {@code logging.properties} file located in this
+   * package. The logging configuration can be overridden in production using the standard {@link LogManager} system
+   * properties "java.util.logging.config.class" or "java.util.logging.config.file".
+   * </p>
+   */
+  public static void initialize() {
+    initialize(LogManager.getLogManager(), System.getProperties());
+  }
+
+  @VisibleForTesting
+  static void initialize(final LogManager logManager, final Properties systemProperties) {
+    if (systemProperties.containsKey(SystemPropertyKeys.LOGGING_CONFIGURATION_CLASS_NAME)
+        || systemProperties.containsKey(SystemPropertyKeys.LOGGING_CONFIGURATION_FILE_NAME)) {
+      return;
+    }
+
+    try (final InputStream is = LoggingConfiguration.class.getResourceAsStream("logging.properties")) {
+      logManager.readConfiguration(is);
```

Comment:
I'm reading the default logging configuration from a properties file for no particular reason.  We could just as well write the configuration in this class as code.  The default configuration is small enough that it wouldn't clutter things up too much.  If, in the future, the configuration gets larger, it could be moved back to a properties file at that time.

Please advise if anyone prefers to not use a properties file for the default configuration.